From 5abd14191c9dc29e783d5e78a4a1783a946c7cda Mon Sep 17 00:00:00 2001
From: Peifeng Yu <peifeng@umich.edu>
Date: Wed, 3 Jan 2018 15:46:38 -0500
Subject: [PATCH] pty: remove deprecated signal constant SIGUNUSED

With glibc 2.6, the obsolete signal constant SIGUNUSED is no longer defined by signal.h.
---
 src/pty.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pty.c b/src/pty.c
index 3494104..1443f4a 100644
--- a/src/pty.c
+++ b/src/pty.c
@@ -299,7 +299,7 @@ static void setup_child(int master, struct winsize *ws)
 	if (ret)
 		log_warn("cannot reset blocked signals: %m");
 
-	for (i = 1; i < SIGUNUSED; ++i)
+	for (i = 1; i < SIGSYS; ++i)
 		signal(i, SIG_DFL);
 
 	ret = grantpt(master);
